### ISSUE TITLE
The API is returning the value of private variables in plain text

### ISSUE THREAD

**Issue by rcicm** [tsuru/tsuru#2356]
Even when accessing events via dashboard (dashboard > events > app.deploy), we can see the content of private variables in plain text.

**Comment by juliobguedes** [comment]
Is there anyone working on this? How can I get started?

**Comment by ericjrsilva** [comment]
Hi guys, I'm trying to figure out what vars i should make private. What i did:

1. I installed tsuru following the doc instructions;
2. I found the function eventInfo in the file event.go in api's folder, It calls GetByID function;
3. I saw that type eventData is the result object when we call eventInfo function;

My question is: what fields are private and what aren't?

I created one test called TestGetEventInfo in event_test.go in api's folder but when i try to run _go test event_test.go_, i got a lot of erros.

`# command-line-arguments [command-line-arguments.test]
./event_test.go:47:19: undefined: nativeScheme
./event_test.go:48:12: undefined: nativeScheme
./event_test.go:51:12: undefined: userWithPermission
./event_test.go:155:12: undefined: RunServer
./event_test.go:165:12: undefined: RunServer
./event_test.go:177:12: undefined: RunServer
./event_test.go:195:12: undefined: RunServer
./event_test.go:207:11: undefined: RunServer
./event_test.go:221:12: undefined: RunServer
./event_test.go:251:12: undefined: RunServer
./event_test.go:251:12: too many errors
FAIL    command-line-arguments [build failed]
FAIL`

cannot i run tests in api's folder?

Thanks!
